.claude/skills/strings-xml-translation-check/scripts/translate-strings-xml.py

```
@@ -0,0 +1,297 @@
+
+
+def parse_strings_xml(path: Path) -> dict[str, str]:
+    """Parse an Android strings.xml. Returns {key: value}.
+
+    - <string name="X">value</string>  -> {"X": value}
+    - <string-array name="X"><item>a</item><item>b</item></string-array>
+      -> {"X[0]": "a", "X[1]": "b"}
+    - Skips elements with translatable="false".
+    - Skips <plurals> and any other element types.
+    """
+    tree = ET.parse(path)
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)